Firestore/core/src/firebase/firestore/remote/grpc_connection.h

```diff
@@ -35,7 +35,10 @@
 #include "grpcpp/channel.h"
 #include "grpcpp/client_context.h"
 #include "grpcpp/completion_queue.h"
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation"
```

Comment:
Does this work in Visual Studio? (by work, I mean, does compilation succeed?) Not that it's a big deal for the release.

Reply:
This does not work for visual studio, but that's not even an experimentally supported environment for the release and I'm specifically trying to avoid a recurrence of issue 2111 in the 1.0.0 release. Let's address on master.